src/mlpack/methods/ann/layer/linear.hpp

```diff
@@ -14,6 +14,8 @@
 #define MLPACK_METHODS_ANN_LAYER_LINEAR_HPP
 
 #include <mlpack/prereqs.hpp>
+#include <mlpack/methods/ann/regularizer/no_regularizer.hpp>
+#include <mlpack/methods/ann/regularizer/regularizer.hpp>
```

Comment:
Since `no_regularizer` is already imported in `regularizer`, I guess line 17 is redundant?